src/rdb/rdb_raft.c

```diff
@@ -122,8 +122,7 @@ rdb_raft_fini_ae(msg_appendentries_t *ae)
 		for (i = 0; i < ae->n_entries; i++) {
 			msg_entry_t *e = &ae->entries[i];
 
-			if (e->data.buf != NULL)
-				D_FREE(e->data.buf);
+			D_FREE(e->data.buf);
```

Comment:
The !NULL check seems like a good idea here.  How come it can be removed?  Does `D_FREE()` do it itself?  I guess it probably should in any case.

Reply:
free() and D_FREE() are both fine with NULL input.  We made a decision a while ago to not compare with NULL before calling, as it improves readability, and adds valuable data to the debug/logging output.  I'm trialing a pre-commit hook that automatically converts any legacy cases when I commit, and I think there are some whitespace changes in this file that come from the same hook.